xml/System.Collections.Immutable/ImmutableArray`1+Enumerator.xml

```diff
@@ -25,7 +25,13 @@
     <summary>An array enumerator.  
   
  **NuGet package**: <see href="https://www.nuget.org/packages/System.Collections.Immutable/">System.Collections.Immutable</see> (<see href="https://docs.microsoft.com/dotnet/api/system.collections.immutable?#remarks">about immutable collections and how to install</see>)</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+      
+It is important that this enumerator does NOT implement <xref:T:System.IDisposable>. We want the iterator to inline when we do foreach and to not result in a try/finally frame in the client.
```

Comment:
```suggestion
> [!IMPORTANT]
> This enumerator should **not** implement <xref:System.IDisposable>. The iterator should inline in a `foreach` loop and not result in a `try/finally` frame in the client.
```
Note that, for remarks, your tool is outputting badly formatted xrefs; the type/member identifier (T, M, F, P, or E) and the colon that follows it should be dropped.

Reply:
Thanks Ron. I fixed the tool, we should not see the type/member identifier in future PRs (but if you do, let me know).